### ISSUE TITLE
Elden Ring ver. 1.03.2

### ISSUE THREAD

**Issue by mapri3500** [Quidrex/DSDeaths#17]
It looks like the script doesnt work with ver. 1.03.2 of Elden Ring (Released on 23th March 2022)

**Comment by Danzoui** [comment]
Works for me. EAC disabled?

**Comment by mapri3500** [comment]
I found my Issue. There are 2 Ways to diasble EAC.
I used this method but it doesnt work with the Script:

Go to the Steam library location where you installed Elden Ring.
Go to steamapps > ELDEN RING > game.
Rename start_protected_game.exe to something else. ...
Make a copy of elden_ring.exe.
Rename this newly made copy to start_protected_game.exe.
Run the game through Steam as normal.

The new method I found was this and it works:

Head over to the folder where the eldenring.exe file is present.
Then right-click in an empty location and select New > Text Document.
Name it steam_appid, double click to open it, and type in the below value: 1245620
Finally, hit the Ctrl+S shortcut keys to save the file.
Now launch the game via the EXE file (and not via Steam).

Maybe this is helpful for other guys with this issue ;)
